src/components/ClusterManager.js

```diff
@@ -363,6 +365,22 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     this.resetUpdateInterval()
   }
 
+  componentDidUpdate(prevProps) {
+    const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
+    const { status, jupyterUserScriptUri } = this.getCurrentCluster() || {}
+    if (status === 'Error' && prevCluster.status !== 'Error') {
```

Comment:
Because the cluster manager re-mounts on each new page, this will cause a new error to pop up on every page transition until the error cluster is removed. We should probably make it less assertive, maybe something like keeping a list of clusters we've notified about, and only generating one notification per cluster.